### PR TITLE
Dashboard modal backend

### PR DESCRIPTION
**Pull request by AaronAlemi**
Dashboard is fully connected to backend. The create modals have been added so you can now directly create new projects/teams from the dashboard, and view your current ones. The ticket statistics are also pulled from the backend but aren't added to the charts yet as we still need to finish the ticket stuff first.

### CODE REVIEW

File: bugtracker-ui/src/components/Tables/DashboardProjectsTable.jsx [hunk 1]
```diff
@@ -51,18 +56,25 @@ const columns = [
 ];
 
 export const DashboardProjectsTable = ({dashboardProjectsModal, setDashboardProjectsModal}) => {
+  const { projects, setProjects, fetchProjects, projectModal, setProjectModal } = useProjectContext()
+  const navigate = useNavigate()
+
+  function onRowClick(data) {
+    console.log("clicked row")
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Tables/DashboardProjectsTable.jsx:63]: Remove/Delete

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comments

**Review by Rebecca-CYLi — APPROVED** [review]
resolve conflicts first

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
oops, there's conflicts

**Comment by AaronAlemi** [comment]
Conflicts resolved and changes implemented

**Pull request merged**